src/lights/PointLight.cpp

```
@@ -0,0 +1,37 @@
+#include "PointLight.hpp"
+#include "Vector3D.hpp"
+#include <cmath>
+#include <algorithm>
+#include <iostream>
+
+Math::Vector3D Raytracer::PointLight::computeLighting(
```

Cppcheck (2.10-2)

[style] The function 'computeLighting' is never used.

(unusedFunction)